forte/data/ontology/configs/stanfordnlp_ontology.json

```diff
@@ -28,27 +24,27 @@
       ]
     },
     {
-      "entry_name": "forte.data.ontology.stanfordnlp_ontology.Sentence",
-      "parent_entry": "forte.data.ontology.top.Annotation",
+      "entry_name": "stanfordnlp_ontology.Sentence",
+      "parent_entry": "Annotation",
       "attributes": [
         {
           "attribute_name": "tokens",
           "attribute_type": "List",
-          "element_type": "forte.data.ontology.stanfordnlp_ontology.Token"
+          "element_type": "stanfordnlp_ontology.Token"
         }
       ]
     },
     {
-      "entry_name": "forte.data.ontology.stanfordnlp_ontology.Document",
-      "parent_entry": "forte.data.ontology.top.Annotation"
+      "entry_name": "stanfordnlp_ontology.Document",
+      "parent_entry": "Annotation"
```

Comment:
We can't accurately find the ``parent_entry`` without a full name, right?

Reply:
If it is a unique name, we can. For example, in this case Annotation is a class in `top`, and doesn't have a duplicate, otherwise, you need to provide the full name.  I will add a test case for (1) full name (2) duplicate entries.

Reply:
It seems that we have to make special cases to deal with this. And users can face errors when they are not careful about this approach. Let's simply use the full qualified name.

Reply:
Changed as discussed. The full name is being used everywhere.